third_party/challenge_bypass_ristretto_cxx/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "challenge-bypass-ristretto-cxx"
+version = "0.1.0"
+authors = ["eV <ev@7pr.xyz>"]
+edition = "2018"
+
+[lib]
+crate-type = [
+    "rlib",
+]
+name = "challenge_bypass_ristretto"
+
+[dependencies]
+challenge-bypass-ristretto = { version = "1.0.1", features = ["base64"] }
```

Comment:
This feature is already enabled in the rust library BUILD.gn. Ok.

Reply:
it is commented in third_party/rust/BUILD.gn:

```
# challenge-bypass-ristretto-ffi
base64 = "0.9.3"
# challenge-bypass-ristretto = { version = "1.0.1", features = ["base64"] }
hmac = "0.10"
rand = "0.7"
# sha2 = "0.9"
```

Reply:
Yes. That should probably be updated for the new crate name, but I guess left commented-out since we don't want the tools to manage it.

I was just confirming the listed features were also enabled in the BUILD.gn for the crate source.